NDTensors/ext/NDTensorsROCmExt/NDTensorsROCmExt.jl

```diff
@@ -0,0 +1,26 @@
+module NDTensorsROCmExt
```

Comment:
This should be called `NDTensorsAMDGPUExt`, following the convention that extensions are named based on the names of the two packages involved.

Reply:
Ah ok, I will change that!